modules/update-network-sysctl-allowlist.adoc

```diff
@@ -0,0 +1,167 @@
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Follow this procedure to modify the predefined list of safe `sysctls`. This procedure describes extending the default allow list, the removal of `sysctls` is also supported.
+
+.Procedure
+
+. View the existing predefined list:
++
+[source,terminal]
+----
+$ oc get cm -n openshift-multus cni-sysctl-allowlist -oyaml
+----
++
+.Expected output
++
+[source,terminal]
+----
+apiVersion: v1
+data:
+  allowlist.conf: |-
+    ^net.ipv4.conf.IFNAME.accept_redirects$
+    ^net.ipv4.conf.IFNAME.accept_source_route$
+    ^net.ipv4.conf.IFNAME.arp_accept$
+    ^net.ipv4.conf.IFNAME.arp_notify$
+    ^net.ipv4.conf.IFNAME.disable_policy$
+    ^net.ipv4.conf.IFNAME.secure_redirects$
+    ^net.ipv4.conf.IFNAME.send_redirects$
+    ^net.ipv6.conf.IFNAME.accept_ra$
+    ^net.ipv6.conf.IFNAME.accept_redirects$
+    ^net.ipv6.conf.IFNAME.accept_source_route$
+    ^net.ipv6.conf.IFNAME.arp_accept$
+    ^net.ipv6.conf.IFNAME.arp_notify$
+    ^net.ipv6.neigh.IFNAME.base_reachable_time_ms$
+    ^net.ipv6.neigh.IFNAME.retrans_time_ms$
+kind: ConfigMap
+metadata:
+  annotations:
+    kubernetes.io/description: |
+      Sysctl allowlist for nodes.
+    release.openshift.io/version: 4.12.0-0.nightly-2022-11-16-003434
+  creationTimestamp: "2022-11-17T14:09:27Z"
+  name: cni-sysctl-allowlist
+  namespace: openshift-multus
+  resourceVersion: "2422"
+  uid: 96d138a3-160e-4943-90ff-6108fa7c50c3
+----
+
+. Edit the list by using the following command:
++
+[source,terminal]
+----
+$ oc edit cm -n openshift-multus cni-sysctl-allowlist -oyaml
+----
++
+For example to allow you to be able to implement stricter reverse path forwarding you need to add `^net.ipv4.neigh.IFNAME.rp_filter$` and `^net.ipv6.neigh.IFNAME.rp_filter$` to the list as shown here:
++
+[source,terminal]
+----
+# Please edit the object below. Lines beginning with a '#' will be ignored,
+# and an empty file will abort the edit. If an error occurs while saving this file will be
+# reopened with the relevant failures.
+#
+apiVersion: v1
+data:
+  allowlist.conf: |-
+    ^net.ipv4.conf.IFNAME.accept_redirects$
+    ^net.ipv4.conf.IFNAME.accept_source_route$
+    ^net.ipv4.conf.IFNAME.arp_accept$
+    ^net.ipv4.conf.IFNAME.arp_notify$
+    ^net.ipv4.conf.IFNAME.disable_policy$
+    ^net.ipv4.conf.IFNAME.secure_redirects$
+    ^net.ipv4.conf.IFNAME.send_redirects$
+    ^net.ipv4.neigh.IFNAME.rp_filter$
+    ^net.ipv6.conf.IFNAME.accept_ra$
+    ^net.ipv6.conf.IFNAME.accept_redirects$
+    ^net.ipv6.conf.IFNAME.accept_source_route$
+    ^net.ipv6.conf.IFNAME.arp_accept$
+    ^net.ipv6.conf.IFNAME.arp_notify$
+    ^net.ipv6.neigh.IFNAME.base_reachable_time_ms$
+    ^net.ipv6.neigh.IFNAME.retrans_time_ms$
+    ^net.ipv6.neigh.IFNAME.rp_filter$
+----
+
+. Save the changes to the file and exit.
+
+.Verification
+
+Follow these steps to verify this added `sysctl` is now available on all nodes.
```

Comment:
A hard proof that this works would be to create a pod with a secondary network that has the new sysctl applied.

Reply:
Hi @mmirecki should I add that note or do I need to revise this verification. I am not sure how that would look, do you have an example.

Reply:
Hi @mmirecki is this an example https://docs.openshift.com/container-platform/4.11/networking/multiple_networks/attaching-pod.html

Reply:
I thought about just saying something like:
`To verify this step create a pod with a secondary network using a network-atachment-definition containing the tuning plugin with the modified sysctl as described <here>.`
<here> - being the link to how to use the sysctls

Reply:
Yes as I have that doc'd elsewhere. I think that occurred to me but then I must have forgot to add it. Thanks @mmirecki for reminding me

Reply:
added the hard proof